templates/addons/linode-cloud-controller-manager-helm.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: linode-cloud-controller-manager
+spec:
+  clusterSelector:
+    matchLabels:
+      ccm: linode
+  repoURL: https://linode.github.io/linode-cloud-controller-manager/
+  chartName: ccm-linode
+  namespace: kube-system
+  options:
+    waitForJobs: true
+    wait: true
+    timeout: 5m
+  valuesTemplate: |
+    secretRef:
+      name: "linode-token-region"
+      apiTokenRef: "apiToken"
+      regionRef: "region"
+    image:
+      tag: v0.3.24
```

Comment:
can we move this version into the version field of the chart? each version should set the image tag correctly